client/README.md

```diff
@@ -0,0 +1,12 @@
+# Frontend
```

Comment:
This is a stub for now. Will add more details in future, but it's expected that only npm/yarn should be required for the final setup.